.kokoro/autogenerate.sh

```diff
@@ -62,9 +62,23 @@ fi
 # (We don't actually need to pack here, but it's simplest to always do so.)
 ./BuildGenerated.sh @tmp/ApisToGenerate.txt
 
+# For each new/updated Discovery file, add the Discovery file
+# and the generated code in a commit.
+for file in $(cat tmp/ApisToGenerate.txt | sed 's/\r//g')
+do
+  service=$(basename $file | sed 's/.json//g')
+  generated=$(dotnet run --project Src/Tools/BuildGeneratedArgumentTranslator -- Src/Generated $file)
```

Comment:
Does this pick up the source file and project file, right?

Reply:
Yup - it's the directory that gets returned, so it'll include both.